Python/bytecodes.c

```diff
@@ -193,7 +191,21 @@ dummy_func(
             ERROR_IF(res == NULL, error);
         }
 
-        inst(BINARY_OP_MULTIPLY_INT, (left, right -- prod)) {
+        family(binary_op, INLINE_CACHE_ENTRIES_BINARY_OP) = {
```

Comment:
Honestly, I don't think we really need the `INLINE_CACHE_ENTRIES_WHATEVER` stuff (or the asserts it produces) in this file anymore (they were originally added to simplify the `JUMPBY(...)` moves, but those are going to be generated now).

I feel like it sort of just complicates parsing and code generation for no real benefit... plus, we actually already have asserts to this effect in `specialize.c` where we ended up re-using these constants in some places.

```suggestion
        family(binary_op) = {
```